portal/src/index.scss

```diff
@@ -0,0 +1,40 @@
+html,
+body {
+  min-height: 100vh;
+}
+
+html {
```

Comment:
IIRC we're going to use fluent design, so these CSS is placeholder right?

Reply:
No. This rules in this file are not placeholder. Let me add more comments to explain the purpose of each rule.